src/StorageAction/StorageAction.Autorest/docs/Get-AzStorageActionTaskAssignment.md

```diff
@@ -14,7 +14,7 @@ Lists Resource IDs of the Storage Task Assignments associated with this Storage
 
 ```
 Get-AzStorageActionTaskAssignment -ResourceGroupName <String> -StorageTaskName <String>
- [-SubscriptionId <String[]>] [-Maxpagesize <String>] [-DefaultProfile <PSObject>] [<CommonParameters>]
+ [-SubscriptionId <String[]>] [-Maxpagesize <Int32>] [-DefaultProfile <PSObject>] [<CommonParameters>]
```

Comment:
Is it better to use MaxPageSize to follow the naming convention?

Reply:
This is autogen code. 
Would you please check with Xiaogang if this is correct?

Reply:
@blueww Maxpagesize is a single word without any separators like hyphen so that it is difficult for autorest to correct it. Directive is the only option to explicitly tell the correct form of this compound word. Since in other modules, it is also -Maxpagesize, we can keep this form in StorageAction.